### PR TITLE
Feat get profile

### PR DESCRIPTION
**Pull request by JakubKostrzewsky**
## Description
Get user profile

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/features/dashboard/presentation/pages/dashboard_page.dart [hunk 1]
```diff
@@ -17,33 +18,45 @@ class DashboardPage extends StatelessWidget {
...
-              ],
-            ),
-            failed: (error) => Text(error.failure.toString()),
-          );
-        },
-      ),
+              ),
+            ],
+          ),
+          failed: (error) => Text(error.failure.toString()),
```
Comment (JakubKostrzewsky) [inline lib/features/dashboard/presentation/pages/dashboard_page.dart:57]: use `FailurePage`

File: lib/features/login/data/repositories/login repository_impl.dart [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+class LoginRepositoryImpl extends LoginRepository {
+  LoginRepositoryImpl(this._client);
+
+  final AppUniteHttpClient _client;
+
+  @override
+  Future<Either<Failure, bool>> login(String serverAuthCode) async {
+    final Response response = await _client.post('/api/v1/oauth/token', body: {"code": serverAuthCode});
+
+    log('TESOTWANIE: ${response.toString()}');
```
Comment (agnkuz) [inline lib/features/login/data/repositories/login repository_impl.dart:18]: do you still need it? 😉

File: lib/features/login/data/repositories/login repository_impl.dart [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import 'dart:developer';
```
Comment (agnkuz) [inline lib/features/login/data/repositories/login repository_impl.dart:1]: please check the file name - it seems that you're missing underscore between login and repository (or just GitHub shows it that way)

Comment (JakubKostrzewsky) [reply lib/features/login/data/repositories/login repository_impl.dart]: Good eye 🔥 👁️  🦅

**Pull request merged**